symplyphysics/core/symbols/quantities.py

```diff
@@ -71,7 +71,7 @@ def _eval_Abs(self) -> Quantity:
 # Allows for some SymPy comparisons, eg Piecewise function
 @dispatch(Quantity, Quantity)  # type: ignore[misc]
 def _eval_is_ge(lhs: Quantity, rhs: Quantity) -> bool:
-    return bool(lhs.scale_factor >= rhs.scale_factor)
+    return lhs.scale_factor >= rhs.scale_factor  # type: ignore[no-any-return]
```

Comment:
Don't silence the mypy checker. I think it hints that scale factor should be evaluated to a number before comparison.